lstchain/scripts/onsite/onsite_create_drs4_time_file.py

```diff
@@ -113,10 +113,10 @@ def main():
         time_file = f"{output_dir}/time_calibration.Run{run:05d}.0000.h5"
 
         print(f"\n--> PRODUCING TIME CALIBRATION in {time_file} ...")
-        cmd = f"lstchain_data_create_time_calibration_file  --input-file {input_file} " \
```

Comment:
Please define the command as list of arguments instead of as string and then calling `split` on it